components/explorer/src/lib.rs

```diff
@@ -12,14 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Blockchain explorer module provides API for getting information about blocks and transactions
+//! Blockchain explorer provides API for getting information about blocks and transactions
```

Comment:
Maybe it makes sense to add entries to both `explorer` component and `explorer` service crate comments/`readme`s explaining how are they different from each other.